logstash-core-event-java/src/main/java/org/logstash/Event.java

```diff
@@ -338,22 +348,33 @@ private Timestamp initTimestamp(Object o) {
             logger.warn("Error parsing " + TIMESTAMP + " string value=" + o.toString());
         }
 
-        tag(TIMESTAMP_FAILURE_TAG);
-        this.data.put(TIMESTAMP_FAILURE_FIELD, o);
-
-        return Timestamp.now();
+        return null;
     }
 
     public void tag(String tag) {
-        List<Object> tags = (List<Object>) this.data.get("tags");
-        if (tags == null) {
+        List<Object> tags;
+        Object _tags = this.getField("tags");
```

Comment:
@colinsurprenant I really don't want to block this PR based on a variable naming suggestion, and I agree it is minor, but this notion of "I wrote code so I'll do what I want" is just antithetical to our review process. 

If it is _that_ minor, why don't you accept it? On the contrary, if we move forward with this, we'll introduce a new convention here that I have not seen used in Java and may permeate our code base. We have a chance to correct it. I can also argue this makes reading code hard. How do I know `_tags` is different than input parameter `tags`. it is so easy to miss when reading this code snippet. 

I do understand we don't have a style guide but I am not sure how this is different than accepting camel case as specified above. And, I look to Elasticsearch codebase as a style reference.